src/glb-healthcheck/TcpHealthChecker.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"expvar"
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"net"
+	"time"
+	 )
+
+var (
+	tcpCounters = expvar.NewMap("TcpHealthChecker")
```

Comment:
Would be good to use this to track counters, helps with tracing what actions this module does, e.g. https://github.com/github/glb-director/blob/master/src/glb-healthcheck/HttpHealthChecker.go#L64

Reply:
Done. Thanks.